test/DGmethods/compressible_Navier_Stokes/rising_bubble-model-imex.jl

```diff
@@ -0,0 +1,220 @@
+# Load Packages 
+using MPI
+using CLIMA
+using CLIMA.Mesh.Topologies
+using CLIMA.Mesh.Grids
+using CLIMA.Mesh.Geometry
+using CLIMA.DGmethods
+using CLIMA.DGmethods.NumericalFluxes
+using CLIMA.MPIStateArrays
+using CLIMA.AdditiveRungeKuttaMethod
+using CLIMA.LinearSolvers
+using CLIMA.GeneralizedMinimalResidualSolver
+using CLIMA.SubgridScaleParameters
+using CLIMA.ODESolvers
+using CLIMA.GenericCallbacks
+using CLIMA.Atmos
+using CLIMA.VariableTemplates
+using CLIMA.MoistThermodynamics
+using CLIMA.PlanetParameters
+using LinearAlgebra
+using StaticArrays
+using Logging, Printf, Dates
+using CLIMA.VTK
+using Random
+using CLIMA.Atmos: vars_state, vars_aux
+
+@static if haspkg("CuArrays")
+  using CUDAdrv
+  using CUDAnative
+  using CuArrays
+  CuArrays.allowscalar(false)
+  const ArrayTypes = (CuArray,)
+else
+  const ArrayTypes = (Array,)
+end
+
+if !@isdefined integration_testing
+  const integration_testing =
+    parse(Bool, lowercase(get(ENV,"JULIA_CLIMA_INTEGRATION_TESTING","false")))
+end
+
+# -------------- Problem constants ------------------- # 
+const (xmin,xmax)      = (0,1000)
+const (ymin,ymax)      = (0,400)
+const (zmin,zmax)      = (0,1000)
+const Ne        = (10,2,10)
+const polynomialorder = 4
+const dim       = 3
+const dt        = 0.01
```

Comment:
This seems to be the same `dt` as the current explicit version of the [rising thermal bubble](https://github.com/climate-machine/CLIMA/blob/9594889aa4b0b461122bc08cc1ee73ccd0eba27c/test/DGmethods/compressible_Navier_Stokes/rising_bubble-model.jl#L47). Can this be made any bigger? (Also don't know if the explicit `dt` is as large as possible.)

Reply:
We need a proper integration test for this, see #462.

Reply:
After adding boundary conditions for the linear model and fixing small problems mentioned in my recent comments I was able to run this test with much bigger time step (e.g. `dt = 0.4` worked).

Reply:
Do you know the largest time step running in fully explicit mode?

Reply:
It would be helpful to be able to be able to extract Courant number values (e.g., both with respect to acoustic waves and velocity). It would also be beneficial to subdivide this information into vertically and horizontally propagating waves.